docs/proposals/pipeline/pipeline.md

```diff
@@ -368,35 +368,55 @@ For more details, see [Tekton signed-provenance-tutorial](https://tekton.dev/doc
 
 #### Kurator Pipeline cli
 
-##### 1. Pipeline Commands
+We are focused on providing the viewing logs of Pipeline executions. Therefore, we have concentrated the CLI's features on this core requirement.
 
-- **`pipeline list`**
-  
-    Display all current Pipelines.
+For viewing information about Pipeline objects, users can directly utilize existing tools like kubectl.
 
-- **`pipeline describe <name>`**
+##### Pipeline Execution Commands
 
-    Show detailed information of a specified Pipeline.
+###### List Commands
 
-- **`pipeline log --lastExecution <name>`**
+- **`kurator pipeline execution list -n`**
 
-    Display the logs of the most recent PipelineExecution triggered by an event for a specified Pipeline.
+This command is used to list all Pipeline execution instances and groups them according to the creator (Pipeline). Each instance will display its creation time and the name of the creator.
 
-##### 2. PipelineExecution Commands
+Here is an example output.
 
-- **`pipeline-execution list [--pipeline=<name>]`**
-  
-    List all PipelineExecution instances. The list includes each execution's event information and timestamp. 
-    
-    If the `--pipeline` parameter is provided, only list PipelineExecutions belonging to a specific Pipeline.
+```console
+-----------------------------------------------------------------------------------
+Execution Name                  | Creation Time          | Namespace        | Pipeline
+-----------------------------------------------------------------------------------
+quick-start-run-frb7l           | 2024-01-02 10:15:30    | kurator-pipeline | quick-start
+quick-start-run-h65wx           | 2024-01-02 11:20:45    | kurator-pipeline | quick-start
+my-app-pipeline-run-a1b2c       | 2024-01-03 14:25:05    | kurator-pipeline | my-app-pipeline
+my-app-pipeline-run-d3e4f       | 2024-01-03 15:55:30    | kurator-pipeline | my-app-pipeline
+my-app-pipeline-run-g6h7i       | 2024-01-03 17:10:00    | kurator-pipeline | my-app-pipeline
+```
 
-- **`pipeline-execution describe <name>`**
+###### Logs Commands
 
-    Show detailed information of a specified PipelineExecution, including task composition, current status.
+- **`kurator pipeline execution logs <execution-name>`**
```

Comment:
For kubectl, it support `kubectl logs`,  here `kurator pipeline logs` ?

Reply:
Yes, it is possible to view the log of a single pod using kubectl.

However, sequentially querying the logs of each container in a group of pods is quite complex. This is precisely what kurator pipeline execution logs offers.